share/instruments/instruments.xml

```diff
@@ -7285,12 +7285,44 @@
                   <genre>concertband</genre>
                   <genre>marching</genre>
             </Instrument>
+            <Instrument id="cajon">
+                  <family>drums</family>
+                  <trackName>Cajón</trackName>
+                  <longName>Cajón</longName>
+                  <shortName>Cjn.</shortName>
+                  <description>Peruvian wooden box drum.</description>
+                  <musicXMLid>drum.cajon</musicXMLid>
+                  <clef>PERC</clef>
+                  <stafftype staffTypePreset="perc1Line">percussion</stafftype>
+                  <barlineSpan>1</barlineSpan>
+                  <drumset>1</drumset>
+                  <singleNoteDynamics>0</singleNoteDynamics>
+                  <Drum pitch="38"> <!--Acoustic Snare-->
+                        <head>normal</head>
+                        <line>0</line>
+                        <voice>0</voice>
+                        <name>Hit</name>
+                        <stem>1</stem>
+                        <shortcut>A</shortcut>
+                        <panelRow>0</panelRow>
+                        <panelColumn>0</panelColumn>
+                  </Drum>
+                  <Channel>
+                        <!--MIDI: Bank 128, Prog 40; MS General: Brush-->
+                        <controller ctrl="0" value="1"/> <!--Bank MSB-->
+                        <program value="40"/> <!--Brush Kit-->
+                  </Channel>
+                  <genre>popular</genre>
+                  <genre>orchestra</genre>
+                  <genre>concertband</genre>
+                  <genre>world</genre>
+            </Instrument>
             <Instrument id="cuica">
                   <family>drums</family>
-                  <trackName>Cuica</trackName>
-                  <longName>Cuica</longName>
+                  <trackName>Cuíca</trackName>
+                  <longName>Cuíca</longName>
                   <shortName>Cu.</shortName>
-                  <description>Cuica.</description>
+                  <description>Brazilian friction drum played by rubbing a stick permanently fastened to the drum membrane on the inside. Applying pressure to the outside increases the pitch.</description>
```

Comment:
Suggested description. Currently, it's too long to fit in the New Score dialog, but I'm hoping that will be resolved in #27891.